src/test/java/ch/cern/eam/wshub/core/services/equipment/TestEquipmentFacadeService.java

```diff
@@ -0,0 +1,27 @@
+package ch.cern.eam.wshub.core.services.equipment;
+
+import ch.cern.eam.wshub.core.services.equipment.entities.Location;
+import ch.cern.eam.wshub.core.tools.InforException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static ch.cern.eam.wshub.core.GlobalContext.*;
+
+/*
+    TEST ASSUMPTIONS:
+        There is a department "HXMF"
```

Comment:
The basic assumptions (like existing entities) could be on the global context because first they could be reused between tests, and secondly to facilitate the state generation for the tests

Reply:
https://github.com/cern-eam/eam-wshub-core/pull/20/commits/d01aa31916d3feca853f9bd9d645931cb8d30cc0